src/components/Stack/Stack.stories.tsx

```diff
@@ -87,37 +87,37 @@ export const ColumnUnits3 = () => (
 export const RowFluid = () => (
   <div
     style={{
-      width: '400px',
-      background: '#f0f0f0',
+      width: "400px",
```

Comment:
we need to add a `.prettierrc` I created a task for this 

https://app.clickup.com/t/8677mxge3

Reply:
yes, sorry for the hum